app/components/Inputs/Label/Label.jsx

```diff
@@ -0,0 +1,14 @@
+// @flow
+import React from 'react'
+import styles from './Label.scss'
+
+type Props = {
+  label: string
+}
+
+export default class Label extends React.Component<Props> {
+  render() {
+    const { label } = this.props
+    return <label className={styles.label}> {label} </label>
```

Comment:
seems there are extra spaces `> { label } <`
also, shouldn't this be a functional component?

Reply:
While there are not extra spaces according to our formatting rules i should expand the API of the component to include additional props like you are suggesting thanks @evgenyboxer was an oversight on my part